app/app.go

```diff
@@ -538,11 +526,8 @@ func NewEvmos(
 	)
 
 	app.EvmKeeper = app.EvmKeeper.SetHooks(
-		evmkeeper.NewMultiEvmHooks(
-			app.RevenueKeeper.Hooks(),
-		),
+		evmkeeper.NewMultiEvmHooks(),
 	)
-
 	// NOTE: app.Erc20Keeper is already initialized elsewhere
```

Comment:
This whole section can be removed as it is a no-op now

Reply:
I left this section intentionally because it will be addressed in a followup PR to remove evm hooks completely